src/humanize/time.py

```diff
@@ -91,16 +91,30 @@ def naturaldelta(value, months=True, minimum_unit="seconds"):
         months (bool): If `True`, then a number of months (based on 30.5 days) will be
             used for fuzziness between years.
         minimum_unit (str): The lowest unit that can be used.
+        when (datetime.timedelta): Point in time relative to which _value_ is
+            interpreted.  Defaults to the current time in the local timezone.
 
     Returns:
         str: A natural representation of the amount of time elapsed.
+
+    Examples
+        Compare two timestamps in a custom local timezone::
+
+        from datetime import datetime, timedelta, timezone
```

Comment:
Please can you change this example to use `import datetime as dt` and `dt.datetime` etc?

Rationale: https://adamj.eu/tech/2019/09/12/how-i-import-pythons-datetime-module/